### PR TITLE
fix(progress-bar): percentage calculation is correct

### LINKED ISSUE #46: Update designs based on user feedback
The latest designs are visible at https://projects.invisionapp.com/share/AF114SEF9385#/screens/453052719.

Notable changes include:

- [ ] Create exit modal to confirm before leaving the wizard (refreshing the page) that all changes will be lost if done
- [x] Make card border optional
  - [x] Defaults to off
- [x] Make card title an h3
- [x] Make card subtitle a p
- [x] Make borders around multi-select/choice optional
  - [x] Defaults to on
- [x] Make question info appear below question component in a summary box
- [x] Make next/prev button configurable
  - [x] Option to left align buttons
  - [x] "Go Back" can be above card
  - [x] Default to text link for "Go Back"
  - [x] Allow config to set default text for all prev/next buttons
- [x] Create 2nd progress bar option for no label design beneath the card

### PR DESCRIPTION
**Pull request by froehlichcr**
Additional fixes #46

- fixes calculation of percent with results and landing steps
- updates storybook custom styles
- adds missing index.ts exports

## Version Bump

- [x] Patch (bug fix - backwards compatible)

### CODE REVIEW

File: src/composable/Questionnaire.ts [hunk 1]
```diff
@@ -172,19 +173,22 @@ export class Questionnaire implements IQuestionnaire {
    * @returns
    */
   getProgressPercent(props: IStepData, config: IQuestionableConfig): number {
+    if (matches(props.step?.sectionId, PAGE_TYPE.RESULTS)) {
+      return 100;
```
Comment (TomNUSDS) [inline src/composable/Questionnaire.ts:177]: 100 or 1.0?

File: src/composable/Questionnaire.ts [hunk 2]
```diff
@@ -172,19 +173,22 @@ export class Questionnaire implements IQuestionnaire {
...
     const thisStepIdx = this.flow.indexOf(`${props.stepId}`);
     // add 2 to account for the summary and result steps
     let lastStepIdx = lastStep.lastStep + 2;
     if (config.mode === MODE.EDIT) {
       // if in design mode, every step will be iterated
-      lastStepIdx = this.flow.length;
+      lastStepIdx = this.flow.length - 1;
     }
     // To calculate the percent, divide the index of this step
     //   by the index of the last step multiplied by 100.
```
Comment (TomNUSDS) [inline src/composable/Questionnaire.ts:194]: Oh, the result is 1-100 but internally the number is 0-1.0?
Line 184 is returning 0.5... so I'm confused.

**Review by TomNUSDS — APPROVED** [review]
Questions about 0-100 vs 0.0-1.0, but otherwise looks good.

File: src/composable/Questionnaire.ts (hunk 1 above)
Comment (froehlichcr) [reply src/composable/Questionnaire.ts:177]: If we're at the results section, we're 100% complete--no further calc required.

**Comment by froehlichcr** [comment]
> Questions about 0-100 vs 0.0-1.0, but otherwise looks good.

The `0.5` is an attempt to make the progress bar smaller when it's at 1% (so less than 1%). Doesn't seem to make a difference, but it doesn't affect the math.

**Pull request merged**